config.m4

```diff
@@ -77,9 +77,14 @@ if test "$PHP_MONGODB" != "no"; then
                  [no])
 
   if test "$PHP_COVERAGE" = "yes"; then
-      PHP_CHECK_GCC_ARG(-fprofile-arcs,                     COVERAGE_CFLAGS="$COVERAGE_CFLAGS -fprofile-arcs")
-      PHP_CHECK_GCC_ARG(-ftest-coverage,                    COVERAGE_CFLAGS="$COVERAGE_CFLAGS -ftest-coverage")
-      EXTRA_LDFLAGS="$COVERAGE_CFLAGS"
+      COVERAGE_CFLAGS="--coverage -g"
```

Comment:
I  also think we should change the option name from --enable-coverage to --enable-mongodb-coverage, in case it clashes with other extensions in static builds.

Reply:
I agree wholeheartedly. I wanted to do that for maintainer flags as well.

Will open new tickets. Should we keep the old options around for BC or are you OK with changing these? They're mainly for internal use, so I don't have a problem with simply renaming them.

Reply:
I'm happy to ditch BC on these.